src/selectors.js

```diff
@@ -65,6 +65,9 @@ Selectors = {
   getUserGroups () {
     return getState().config.get('userGroups')
   },
+  getPortalPage () {
```

Comment:
Humm, I see the `getCurrentPage` is set for `page`.  I'd like to see `getPortalPage` renamed `getCurrentPage` and then rename the old `getCurrentPage` to `getCurrentFetchPage` or similar.  That way the codebase will be consistent on naming.